### ISSUE TITLE
Remove noCache from "mainline" Helmet

### ISSUE THREAD

**Issue by EvanHahn** [helmetjs/helmet#215]
In an effort to focus this project's scope, I'm trying to make sure that Helmet only focuses on HTTP response headers that are strictly related to security. While `nocache` is useful, it's not strictly security-related in my eyes, so I'm removing it.

**This functionality isn't going away. It's just moving to a separate module, `nocache`. You can find it [on npm](https://www.npmjs.com/package/nocache) and [on GitHub](https://github.com/helmetjs/nocache).**

Instead of this:

```js
const helmet = require('helmet')
// ...
app.use(helmet.noCache())
```

You'll do this:

```js
const noCache = require('nocache')
// ...
app.use(noCache())
```

This is a breaking change that will be part of Helmet 4.

**Comment by unbalanced** [comment]
Hey, can you refer me to information on why it was removed from the "default" settings and will be removed in Helmet 4?

Thanks

**Comment by EvanHahn** [comment]
In an effort to focus this project's scope, I'm trying to make sure that Helmet only focuses on HTTP response headers that are strictly related to security. While `nocache` is useful, it's not strictly security-related in my eyes, so I'm removing it.

This functionality isn't going away. It's just moving to a separate module, `nocache`.

**Comment by JEstradaDev** [comment]
hi!! could you add the link for `nocache` module or new documentation?? please

thanks.

**Comment by EvanHahn** [comment]
@JEstradaDev Good idea. I've added more information to the description of this issue.

**Comment by Paul75** [comment]
When I do like this :
const noCache = require('nocache')
// ...
app.use(noCache())

and remove in the securityNoCacheMiddleware the .noCache() that when I do request get error "alreadt set header ...."

thanks

**Comment by EvanHahn** [comment]
This was done in ca483bdd0cada1c2304dc4a76d7eca6cbbb06656 and merged into the upcoming v4 branch (see #197).
